deepforest/utilities.py

```diff
@@ -569,71 +571,72 @@ def project_boxes(df, root_dir, transform=True):
     return df
 
 
-def download_ArcGIS_REST(url,
-                         xmin,
-                         ymin,
-                         xmax,
-                         ymax,
-                         savedir,
-                         additional_params=None,
-                         image_name="image.tiff",
-                         download_service="exportImage"):
+async def download_ArcGIS_REST(semaphore, limiter, url, xmin, ymin, xmax, ymax, bbox_crs, savedir, additional_params=None, image_name="image.tiff", download_service="exportImage"):
+    
     """
-    Fetch data from a web server using geographic boundaries. The data is saved as a GeoTIFF file. The bbox is in the format of xmin, ymin, xmax, ymax for lat long coordinates..
-    This function is used to download data from an ArcGIS REST service, not WMTS or WMS services. 
+    Fetch data from a web server using geographic boundaries and save it as a GeoTIFF file.
+    This function is used to download data from an ArcGIS REST service, not WMTS or WMS services.
     Example url: https://gis.calgary.ca/arcgis/rest/services/pub_Orthophotos/CurrentOrthophoto/ImageServer/
-
-    There are many types of ImageServer, mapServer and open source alternatives. This function is designed to work with ArcGIS ImageServer, but may work with other services. Its very hard to anticipate all params and add to additional_params and download_service name to meet the specifications.
+    
```

Comment:
Given the complexity of these args, we should have a mock example in the docstring and link to the annotation readthedoc page, most users won't be familiar with the asyncio object.